File: 11_November_Batch/C/lcm.c
#include<stdio.h>
int lcm(int a,int b){
    if(a==b){
        return a;
    }
    int max;
    if (a>b){
        max=a;
    }
    else{
        max=b;
    }
    int i=max;
    while (1)
    {
        if (max%a==0&&max%b==0)
        {
            return max;
        }
        max=max+i;
        
    }
    
}
int main(){
    int a,b,ans;
    scanf("%d%d",&a,&b);
    ans=lcm(a,b);
    printf("%d",ans);
    return 0;
}

File: 11_November_Batch/Python/lcm.py
def lcm(a, b):
    if a == b:
        return a
    if a < b:
        max_value = b
    if b < a:
        max_value = a

    i = max_value
    while True:
        if max_value % a == 0 and max_value % b == 0:
            return max_value
        max_value += i


a = int(input("Enter first number: "))
b = int(input("Enter first number: "))
ans = lcm(a,b)
print(ans)
